package.json

```diff
@@ -14,10 +14,13 @@
     "bootstrap": "^4.4.1",
     "core-js": "^3.6.4",
     "jquery": "^3.4.1",
+    "pluralize": "^8.0.0",
     "popper.js": "^1.16.1",
     "register-service-worker": "^1.6.2",
+    "sweetalert2": "^9.10.12",
```

Comment:
Ok, which library do you suggest? I choose that library because is highly configurable and its license is *MIT*, allowing us to use without legal problems.

Reply:
I'd review it and looks good. We can try it.